libvirt/install_operator.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+#
+# Copyright Confidential Containers Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# Install the CAA operator in a Kubernetes cluster.
+#
+set -o errexit
+set -o nounset
+set -o pipefail
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+LIBVIRT_IP="${LIBVIRT_IP:-}"
+LIBVIRT_USER="${LIBVIRT_USER:-$USER}"
+LIBVIRT_NET="${LIBVIRT_NET:-default}"
+LIBVIRT_POOL="${LIBVIRT_POOL:-default}"
+SSH_KEY_FILE="${SSH_KEY_FILE:-}"
```

Comment:
would it make sense to generate temporary one for the user?

Reply:
Yeah, I was considering that.... for CI I will need to automatically create it. For the general case I was not sure. It seems a good idea for me. What about you?

Reply:
personally i crate a new key pair rather than passing the my default private key around (which i used for many other things)